go.mod

```diff
@@ -1,10 +1,12 @@
 module oras.land/oras-go
 
-go 1.22
+go 1.22.4
```

Comment:
Can it be `go 1.22.0`?

Reply:
[github.com/distribution/distribution/v3](https://github.com/distribution/distribution) requires go version >= `1.22.4`.